server/fleet/teams.go

```diff
@@ -156,6 +156,13 @@ type TeamWebhookSettings struct {
 	FailingPoliciesWebhook FailingPoliciesWebhookSettings `json:"failing_policies_webhook"`
 }
 
+type TeamSpecSoftware struct {
+	URL               string `json:"url"`
+	PreInstallQuery   string `json:"pre_install_query"`
+	InstallScript     string `json:"install_script"`
+	PostInstallScript string `json:"post_install_script"`
```

Comment:
Just now realizing that as I started looking at it for gitops, the install/post-install scripts should bean object with a "path" sub-key based on figma: https://www.figma.com/design/oQl2oQUG0iRkUy0YOxc307/%2314921-Deploy-security-agents-to-macOS%2C-Windows%2C-and-Linux-hosts?node-id=822-14330&t=7mEAuAPggWIhTdBM-0 :(

Reply:
thanks! I'll update this now unless you're already started, just lmk!

Reply:
I haven't yet! As mentioned on slack, if you're not feeling good don't hesitate to get some rest and I can take care of it after lunch.

Reply:
okay, this was simpler than expected. I think it's done (after manually testing, and seems alright)